### PR TITLE
LPD-38641 [Test Fix] ObjectActionLocalServiceTest

### PR DESCRIPTION
**Pull request by icgsbr**
# Jira Link
- https://liferay.atlassian.net/browse/LPD-38641
# Additional Information
- On the first [commit](https://github.com/liferay-bpm/liferay-portal/pull/3751/commits/32cea9746bf6877e229c67faf77ac7ae100cdd30), I tried to adjust the code to the pattern we already use in our tests by getting an admin user using `UserTestUtil.getAdminUser` instead of adding one more with `UserTestUtil.addOmniadminUser()`. I followed the examples from other objects tests classes.

### CODE REVIEW

**Comment by icgsbr** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
Hey @carolmariaabb 
Could you review this PR, please? :smile:

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
@icgsbr the `ObjectActionLocalServiceTest.testConcurrentObjectActions` is still failing. Could you please take a look?

**Comment by icgsbr** [comment]
ci:test:sf

**Comment by icgsbr** [comment]
ci:test:relevant

**Comment by icgsbr** [comment]
ci:test:sf

**Comment by icgsbr** [comment]
ci:test:relevant

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java [hunk 1]
```diff
@@ -219,6 +219,11 @@ public void setUp() throws Exception {
 					0, ObjectFieldConstants.BUSINESS_TYPE_TEXT, null,
 					ObjectFieldConstants.DB_TYPE_STRING, true, false, null,
 					"Last Name", "lastName", false, true)));
+
+		_objectDefinition.setName("A" + RandomTestUtil.randomString());
+
+		_objectDefinitionLocalService.updateObjectDefinition(_objectDefinition);
+
```
Comment (carolmariaabb) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java:226]: @icgsbr I don't understand this change. 

The `ObjectDefinitionTestUtil.addCustomObjectDefinition` is already generating a random name with the same pattern: see [here](https://github.com/carolmariaabb/liferay-portal/blob/368c8f1740915dd0a022a092aed686ad5acccd06/modules/apps/object/object-test-util/src/main/java/com/liferay/object/test/util/ObjectDefinitionTestUtil.java#L42) and [here](https://github.com/carolmariaabb/liferay-portal/blob/368c8f1740915dd0a022a092aed686ad5acccd06/modules/apps/object/object-test-util/src/main/java/com/liferay/object/test/util/ObjectDefinitionTestUtil.java#L143).

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

**Comment by carolmariaabb** [comment]
@icgsbr as discussed, I've createad a new subtask to handle the other test failure: https://liferay.atlassian.net/browse/LPD-40469

Please see https://github.com/brianchandotcom/liferay-portal/pull/155582

**Pull request closed without merging**